app/views/users/dossiers/show/_estimated_delay.html.haml

```diff
@@ -0,0 +1,9 @@
+/ FIXME: remove the custom procedure switch at some point
```

Comment:
Ça ne serait pas l’occasion? Le critère pour “at some point” n’est pas clair, le message de commit original de gregoirenovel disait “Hide the mean verification/instruction times for a special demarche  It is not relevant for this demarche, and we’re doing them a favor”, et ce n’est plus la période des stages de troisième.